### ISSUE TITLE
Training beyond specified 't_total'. Learning rate multiplier set to 0.0. Please set 't_total' of WarmupLinearSchedule correctly.

### ISSUE THREAD

**Issue by akogsie** [neuspell/neuspell#26]
I have the following error when I try to run checker.finetune() on my dataset. Any idea why ?

Training beyond specified 't_total'. Learning rate multiplier set to 0.0. Please set 't_total' of WarmupLinearSchedule correctly.

**Comment by murali1996** [comment]
Hi @akogsie, I think this can happen when you have the number of fine-tuning samples less than the batch size. Can you provide more details on those numbers so that we can debug the issue better? Thanks.

**Comment by ahmedbr** [comment]
> Hi @akogsie, I think this can happen when you have the number of fine-tuning samples less than the batch size. Can you provide more details on those numbers so that we can debug the issue better? Thanks.

I'm having the same issue. My numbers are as follow:

train_data: 40000
TRAIN_BATCH_SIZE: 8
GRADIENT_ACC: 4
N_EPOCHS: 2
t_total: 2500

But training goes beyond t_total=2500 value.
